internal/lib/container_server.go

```diff
@@ -194,7 +194,12 @@ func (c *ContainerServer) LoadSandbox(id string) (retErr error) {
 		return errors.Wrapf(err, "error unmarshalling %s annotation", annotations.NamespaceOptions)
 	}
 
-	sb, err := sandbox.New(id, m.Annotations[annotations.Namespace], name, m.Annotations[annotations.KubeName], filepath.Dir(m.Annotations[annotations.LogPath]), labels, kubeAnnotations, processLabel, mountLabel, &metadata, m.Annotations[annotations.ShmPath], m.Annotations[annotations.CgroupParent], privileged, m.Annotations[annotations.RuntimeHandler], m.Annotations[annotations.ResolvPath], m.Annotations[annotations.HostName], portMappings, hostNetwork)
+	created, err := time.Parse(time.RFC3339Nano, m.Annotations[annotations.Created])
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return errors.Wrap(err, "parsing created timestamp annotation")
```

Reply:
Error wrapping is done now. Working on a test now (which might be a little more involved ...)

Reply:
> Working on a test now (which might be a little more involved ...)

Actually, creating the test was easy. Getting the test suite to run locally was somewhat more complicated ;-)

@saschagrunert I've added the testcase to one of the existing tests in `restore.bats` which seemed to be a good fit. If you prefer I can also move it to a separate test in that file.